pycytominer/cyto_utils/load.py

```diff
@@ -169,13 +176,18 @@ def load_npz_locations(npz_file):
     except FileNotFoundError:
         return pd.DataFrame([])
 
-    files = npz.files
+    # number of columns with data in the locations file
+    location_cols = npz["locations"].shape[1]
+    # throw error if user tries to index columns that don't exist
+    if location_x_col_index >= location_cols:
```

Comment:
Is there a reason to enter line 193 if the location_x and y throw outofbounds errors? Or, is line 193 meant for another purpose?

in other words, what situations would returning an empty dataframe be optimal?

Reply:
Line 193 no longer has a purpose and we can't think of any situations in which returning an empty dataframe would be optimal. Removed in https://github.com/cytomining/pycytominer/pull/218/commits/0188aa4f73130a27afa574596ff6f199ada5be0d